Shape_detection/doc/Shape_detection/Shape_detection.txt

```diff
@@ -500,23 +512,26 @@ The `NeighborQuery` related class does not require any parameters, because edge-
 are found using the internal face graph connectivity, while the `RegionType` related class depends on three parameters:
 
 - `distance_threshold` - the maximum distance from the furthest face vertex to a plane;
-- `angle_threshold` - the maximum accepted angle between the face normal and the normal of a plane;
+- `angle_deg_threshold` - the maximum accepted angle between the face normal and the normal of a plane;
```

Comment:
remove `deg`?

Reply:
First, I did not have `deg` but then I noticed that some people when using the code provided radian values so I changed to deg to be sure that people do not miss that the angle is in degrees.

Reply:
Should I still remove it?

Reply:
You can put that in the text.